framework/src/userobject/UserObject.C

```diff
@@ -44,12 +44,16 @@ UserObject::validParams()
   params.addParam<bool>("force_postaux", false, "Forces the UserObject to be executed in POSTAUX");
   params.addParam<bool>(
       "force_preic", false, "Forces the UserObject to be executed in PREIC during initial setup");
+  params.addParam<int>("execution_order_group",
+                       0,
```

Comment:
So, developers are allowed to set a negative value?

Reply:
Yes, negative values are useful to have a single uo execute before all other objects with a default of zero